sisyphus/simple_linux_utility_for_resource_management_engine.py

```diff
@@ -204,6 +210,8 @@ def submit_helper(self, call, logpath, rqmt, name, task_name, start_id, end_id,
         :param int end_id:
         :param int step_size:
         """
+        if self.job_name_mapping is not None:
```

Comment:
You are now repeating the same three lines twice. It would be better to have method which calls self.job_name_mapping (if it's not None) and does the escaping.

You can delete the escape_name function afterwards since it's not needed anymore.